src/annotator/config/test/index-test.js

```diff
@@ -51,24 +67,117 @@ describe('annotator/config/index', function () {
     });
   });
 
-  ['appType', 'openSidebar', 'subFrameIdentifier'].forEach(function (
-    settingName
-  ) {
-    it(`reads ${settingName} from the host page, even when in a browser extension`, function () {
-      getConfig('all', 'WINDOW');
-      assert.calledWith(
-        fakeSettingsFrom().hostPageSetting,
-        settingName,
-        sinon.match({ allowInBrowserExt: true })
-      );
+  describe('browser extension', () => {
+    context('browser extension set to true', function () {
```

Comment:
```suggestion
    context('when client is loaded from the browser extension', () => {
```

The previous naming suggested there was a flag somewhere that is set to a boolean value indicating if this is the browser extension, but that's not how it works.

I rephrased this in a way that better describes the situation rather than how we check for that situation.